docs/README.md

```diff
@@ -228,88 +228,100 @@ Structure the `search-result-layout` or the `search-result-layout.customQuery`,
+|       `undefined`       |      `block`       | Defines which blocks should be rendered per layout. **Caution**: this prop _name_ is not `undefined`. Instead, its name must be the value passed to the `component` prop. This prop's value, in turn, must match the block name of your choosing to be rendered in that specific layout. Check out the example below in order to understand the underlying logic behind this prop. |     `undefined`     |
+| `customSummaryInterval` |      `number`      | Defines the item interval at which the Gallery should render a custom `product-summary` block. For example: declaring `5` would render a custom block at every 4 items rendered, as shown [here](https://user-images.githubusercontent.com/1207017/101687291-0cff1780-3a49-11eb-9c00-678b70001c8a.jpg). _Caution_: This prop doesn't support `layouts` yet.                        |     `undefined`     |
+|     `CustomSummary`     |      `block`       | Defines a block to be rendered according to the interval defined by the `customSummaryInterval` prop.                                                                                                                                                                                                                                                                              |     `undefined`     |
+|     `preferredSKU`      | `PreferredSKUEnum` | Controls which SKU will be selected in the summary                                                                                                                                                                                                                                                                                                                                 | `"FIRST_AVAILABLE"` |
+
+For `PreferredSKUEnum`:
+
+| Name            | Value             | Description                                        |
+| --------------- | ----------------- | -------------------------------------------------- |
+| First Available | `FIRST_AVAILABLE` | Selects the first available SKU in stock it finds. |
+| Last Available  | `LAST_AVAILABLE`  | Selects the last available SKU in stock it finds.  |
+| Cheapest        | `PRICE_ASC`       | Selects the cheapest SKU in stock it finds.        |
+| Most Expensive  | `PRICE_DESC`      | Selects the most expensive SKU in stock it finds.  |
+
+⚠️ Keep in mind that the default behaviour for the initially selected SKU, has now changed to a Product (field) specification.  
+If that one doesn't exist, it will use the `preferredSKU` prop. You can read more about it, and how to implement it in [Recipes](https://vtex.io/docs/recipes/all)
```

Comment:
Same as https://github.com/vtex-apps/product-summary/pull/328#discussion_r689766102

Reply:
We already talked with the education team, they told us they will create the recipe once the PR is merged. I can come back and update the readme with the specific URL once we have it; what do you think?